hpat/hiframes/hiframes_typed.py

```diff
@@ -177,8 +177,8 @@ def _run_assign(self, assign):
             self._type_changed_vars.append(lhs)
 
         if isinstance(rhs, ir.Expr):
-            if rhs.op == 'getattr':
-                return self._run_getattr(assign, rhs)
+            # if rhs.op == 'getattr':
```

Comment:
As far as I understand this handles diffrent input types. Not only Series. If you like to remove it for Series you need to implement type check here (`not isinstance(rhs.something, SeriesType`)